src/components/box/box.test.tsx

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import { getGapValue } from "style/utils/box-gap";
+import { render, screen } from "@testing-library/react";
+import "@testing-library/jest-dom/extend-expect";
+import {
+  testStyledSystemSpacing,
+  testStyledSystemColor,
+  testStyledSystemLayout,
+  testStyledSystemFlexBox,
+  testStyledSystemGrid,
+  testStyledSystemPosition,
+} from "../../__spec_helper__/test-utils";
+import Box from "./box.component";
+import Logger from "../../__internal__/utils/logger";
+
+// mock Logger.deprecate so that no console warnings occur while running the tests
+const loggerSpy = jest.spyOn(Logger, "deprecate");
```

Comment:
**suggestion**: what do you think about removing this mock, so any legitimate warnings that the tests throw that need addressing aren't masked?

Reply:
I think it's worth trying to mock it for the specific test we want to ignore the console for, yes there's a small chance that specific one will have another legitimate console warning it's not likely a huge issue and removing the mock altogether means we'll have a lot of superfluous console logging polluting the output when we run our tests

Reply:
Actually, is there any deprecation warnings we need to mock here?

Reply:
I guess that's actually hangover from when we had deprecation warnings on Box. So I'm happy to remove it.